src/read/macho/dyld_cache.rs

```diff
@@ -241,6 +760,44 @@ where
     }
 }
 
+impl<E: Endian> macho::DyldCacheMappingAndSlideInfo<E> {
+    /// Return the (optional) array of slide information structs
+    pub fn slide<'data, R: ReadRef<'data>>(
+        &self,
+        endian: E,
+        data: R,
+    ) -> Result<Option<DyldCacheSlideInfoSlice<'data, E>>> {
+        match self.slide_info_file_size.get(endian) {
+            0 => Ok(None),
+            _ => {
+                let slide_info_file_offset = self.slide_info_file_offset.get(endian);
+                let version = data
+                    .read_at::<U32<E>>(slide_info_file_offset)
+                    .read_error("Invalid slide info file offset size or alignment")?
+                    .get(endian);
+                match version {
+                    5 => {
+                        let slide = data
+                            .read_at::<macho::DyldCacheSlideInfo5<E>>(slide_info_file_offset)
+                            .read_error("Invalid dyld cache header size or alignment")?;
+                        let page_starts_offset = slide_info_file_offset
+                            .checked_add(mem::size_of::<macho::DyldCacheSlideInfo5<E>>() as u64)
+                            .unwrap();
+                        let page_starts = data
+                            .read_slice_at::<U16<E>>(
+                                page_starts_offset,
+                                slide.page_starts_count.get(endian) as usize,
+                            )
+                            .read_error("Invalid dyld cache header size or alignment")?;
+                        Ok(Some(DyldCacheSlideInfoSlice::V5(slide, page_starts)))
+                    }
+                    _ => todo!(),
```

Comment:
`todo()` can take a message for when it gets hit & panics, e.g. `todo("handle other dcs versions")`